### ISSUE TITLE
Add cable flavour and has_whitening_filter to rfinput struct

### ISSUE THREAD

**Issue by gsleap** [MWATelescope/mwalib#64]
From @andreww5au:
```
Every cable flavour starting with 'RG6', except for 'RG6_90' and 'everything starting with LMR' [has a whitening filter]. 
```

1. We want to add another field for each rf_input which is "flavour" which is just the string that AW provides in the metafits?
2. Add a new bool "has_whitening_filter". TRUE if ((cable_flavour starts with "RG6" AND cable_flavour != "RG6_90") OR cable_flavour starts with "LMR")

**Comment by d3v-null** [comment]
relevant ticket in Birli https://github.com/MWATelescope/Birli/issues/147

**Comment by gsleap** [comment]
Fixed by 22547ed472909ae0fd73ff52b5d99aea6714caad
